.github/workflows/build.yml

```diff
@@ -87,6 +95,29 @@ jobs:
         docker push antrea/antrea-ubi:latest
         docker push antrea/antrea-agent-ubi:latest
         docker push antrea/antrea-controller-ubi:latest
+    - name: Install Kind
+      run: |
+        KIND_VERSION=$(head -n1 ./ci/kind/version)
+        curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-$(uname)-amd64
+        chmod +x ./kind
+        sudo mv kind /usr/local/bin
+    - name: Run basic e2e tests
```

Comment:
I would agree Antonin to put this kind of e2e test to a file `kind_ubi.yml` which should be similar like `kind.yml`.

Reply:
Created https://github.com/antrea-io/antrea/issues/6031 to track the follow-up issue.